python/dgl/contrib/sampling/sampler.py

```diff
@@ -436,6 +436,45 @@ def fetch(self, current_nodeflow_index):
         nflows = [NodeFlow(self.g, obj) for obj in nfobjs]
         return nflows
 
+class EdgeSubgraph(subgraph.DGLSubGraph):
+    ''' The subgraph sampled from an edge sampler.
+
+    A user can access the head nodes and tail nodes of the subgraph directly.
```

Comment:
Maybe give a concrete subgraph example and what's the output of head_nid and tail_nid?

Reply:
Actually, such example is better in the docstring of EdgeSampler.